test/ANNOTATIONS.yaml

```diff
@@ -1043,7 +1043,7 @@ memleaksfull:
   03/09/19:
     - Close user leaks in linked list tests (#12531)
   03/12/19:
-    - Close a memory leak in linkedlist test (#12551)
+    - Close a memory leak in linkedList test (#12551)
```

Comment:
Note that this is just taken from the PR title, so I think I'd leave this one as-is

Reply:
The file modified by the PR is named `linkedList.chpl` though.

Reply:
...but if the annotations file is supposed to list the PR title, I can undo this part.